src/test/ui/rfc-2361-dbg-macro/dbg-macro-expected-behavior.rs

```diff
@@ -54,6 +54,17 @@ fn test() {
         7331
     }));
     assert_eq!(foo, 42);
+
+    // Test trailing comma:
+    assert_eq!("Yeah", dbg!("Yeah", ));
```

Comment:
The result of `dbg!(x,)` is of type `(typeof(x),)`. IOW, a 1-element tuple is produced.

Reply:
Yep, I just changed the test to that effect.